contracts/sweep/SweepERC721.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.0;
+
+import "./TokenSweep.sol";
+
+import "@openzeppelin/contracts-upgradeable/token/ERC721/IERC721Upgradeable.sol";
+
+/**
+ * @title Adds the ability to sweep a token to a beneficiary address
+ *
+ * @dev
+ */
+abstract contract SweepERC721 is TokenSweep {
+    /**
+     * @notice Sweep the erc721 tokens to the beneficiary address
+     *
+     * @dev Needs access control implemented in the inheriting contract
+     **/
+    function sweepERC721Tokens(address token, uint256 tokenId)
```

Comment:
The same decision outcome for SweepERC20 applies here too

Reply:
Just the point you raised about the function being internal and it's good to go 👍